R/utilities-data.R

```diff
@@ -17,7 +17,7 @@ readOSPSTimeValues <- function(dataConfiguration) {
 
   observedData <- list()
   for (sheet in dataConfiguration$dataSheets) {
-    data <- readxl::read_excel(path = filePath, sheet = sheet)
+    data <- readxl::read_excel(path = filePath, sheet = sheet, .name_repair = ~ vctrs::vec_as_names(..., repair = "unique", quiet = TRUE))
```

Comment:
@hannaei `read_excel` throws annoying messages when there are e.g. some excel cells that do not have a proper column name (values "outside" of the required excel structure). I found a workaround and added it to the wiki
https://github.com/esqLABS/esqlabsR/wiki/R-useful-information

Very annoying, but we have to think about it every time we use read_excel (or find a better solution for it).